### PR TITLE
Django 1.11

### PR DESCRIPTION
**Pull request by Ian-Foote**

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -45,26 +45,3 @@ def test_isolate_method():
...
-    assert direct
-    assert not m2m
-
-
-def test_get_field_by_name_many_related():
-    field, model, direct, m2m = utils.get_field_by_name(ManyRelated, 'relation')
-    assert field == ManyRelated._meta.get_field('relation')
-    assert not model
-    assert direct
-    assert m2m
```
Comment (meshy) [inline tests/test_utils.py]: Are these tests no longer worthwhile?

File: .travis.yml [hunk 2]
```diff
@@ -9,8 +9,8 @@ env:
   global:
     - DATABASE_URL='postgres://postgres@localhost/test_utils'
   matrix:
-    - DJANGO='django>=1.9,<1.10' DRF='djangorestframework>=3.0.5,<3.4'
-    - DJANGO='django>=1.8,<1.9' DRF='djangorestframework>=3.0.5,<3.4'
```
[the remark this answers is not in the document]
Comment (Ian-Foote) [reply .travis.yml]: Good catch!

File: requirements.txt [hunk 3]
```diff
@@ -1,9 +1,10 @@
--e .
+-e .[images]
 django>=1.8,<1.9
```
[the remark this answers is not in the document]
Comment (Ian-Foote) [reply requirements.txt]: I'm not sure either - I'd like to delete this as a legacy format, but I'm not sure how to handle the `django` version in the travis matrix.

[the remark this answers is not in the document]
Comment (meshy) [reply requirements.txt]: Would something like this work?

```
pipenv install
pipenv install --selective-upgrade $DJANGO
```

**Pull request merged**